### ISSUE TITLE
mpca_* (AST-related) functions are not documented

### ISSUE THREAD

**Issue by sphynx** [orangeduck/mpc#97]
It would be nice to provide some examples of traversing the AST, describing its structure, etc. At least something similar to [the chapter in "Build your own LISP"](http://www.buildyourownlisp.com/chapter7_evaluation#trees)

**Comment by orangeduck** [comment]
Yes I think we can add some documentation for these. I'll try to look into it or you can make a pull request.
